dissect/target/plugins/apps/vpns/wireguard.py

```diff
@@ -0,0 +1,150 @@
+"""
+    WireGuard configuration parser.
+
+    Resources:
+    - https://manpages.debian.org/testing/wireguard-tools/wg.8.en.html#CONFIGURATION_FILE_FORMAT
+    - https://github.com/pirate/wireguard-docs
+"""
```

Comment:
Could you move this to the docstring of the `WireGuardPlugin` class? Then it will also show up in the help messages etc.